### DISCUSSION TITLE
新型 AI 詐騙

### DISCUSSION THREAD

**Discussion by Tzuhsuannnnn** [1101-GS4719-C/Python-Community-GS4719-C#1497] [General]
大家應該都有多多少少聽過運用AI來犯案的案例，像是之前台灣YOUTUBE圈很有名的事件-小玉透過deepfake 將女藝人、網紅等女性的臉部惡意合成到成人影片上，主要都是透過深度學習和影像合成，替換操縱現有的媒體內容。涉及到假訊息散布、誹謗以及侵犯個人隱私等罪刑。

但隨著科技的進步，現在的AI詐騙已經進階到另一種領域了，例如前陣子在中國發生的AI詐騙事件-詐騙公司利用AI換臉技術打視訊電話給一個中國科技業老闆，取得信任後詐取了 1876萬。

我認為這樣的轉變其實非常恐怖，一般人在接到視訊電話時通常不太會去思考是否為真人，因為在我們原本的認知中只有本人才有辦法以相同面孔、聲音出現在視訊電話中。沒想到現在還需要思考視訊另一端是人類還是機器偽造出來的。
若是這項技術繼續發展，可能會有更恐怖的AI詐騙出現...

相關報導參考:https://udn.com/news/story/7332/7182429

**Comment by Superlan8** [comment]
而且不只AI換臉技術詐騙，現在AI還能模仿別人的語音，再加上換臉技術，AI越做越像真人，可想而知，更高階詐騙也會隨之而來，因此我認為要應對這件事的話，法律也就顯得更重要了。

**Comment by Alexgns** [comment]
新型AI詐騙手法正不斷出現，對毫無戒心的人造成威脅。保持警惕，了解詐騙者使用的最新技巧，保護個人信息和財務安全至關重要。實施強大的安全措施，與基於AI的平台互動時要謹慎小心，可以幫助減輕與這些詐騙行為相關的風險。

**Comment by zhiron544** [comment]
其實我覺得這只是當前大眾對於AI的理解不夠深入，當過一段時間後，這種詐騙自然會逐漸減少。

**Comment by poweichen00** [comment]
我覺得之後肯定會有更多使用AI相關的詐騙，詐騙的技術也越來越好，只能平常多注意不要被詐騙。

**Comment by Chiuhsianglin** [comment]
時常在youtube看到很多人分享自己被詐騙的經驗，我也從來沒有想過自己會被詐騙，希望沒有這一天，而我認為如果我們不管遇到這種新型詐騙還是典型的，都要再三確認，防止自己的錢飛走。

**Comment by PaulSu0905** [comment]
因為這樣的技術讓我在網路上看影片時都會懷疑，那些直播網紅或網帥真的長那樣嗎，會不會也是換臉下的產物，如果是的話，那他所製造的營利行為算是詐騙嗎?

**Comment by lin157644** [comment]
很遺憾看到AI學習能力被運用在犯罪用途上，雖然AI語音生成與換臉已經出世一段時間了，但是剛推出時很多憧憬是讓家人讓逝去親人的聲音等等正面的目標，我覺得AI本身就是工具，而工具本就是一體兩面的，目前看來也只能透過法律與道德約束AI的使用者。
